recipe/build.sh

```diff
@@ -8,10 +8,10 @@ unset CFLAGS
 # Depending on our platform, shared libraries end with either .so or .dylib
 if [[ `uname` == 'Darwin' ]]; then
      export LIBRARY_SEARCH_VAR=DYLD_FALLBACK_LIBRARY_PATH
-#    DYLIB_EXT=dylib
+     DYLIB_EXT=dylib
 else
      export LIBRARY_SEARCH_VAR=LD_LIBRARY_PATH
-#    DYLIB_EXT=so
+     DYLIB_EXT=so
```

Comment:
So I don't think we need these any more. One can use [`SHLIB_EXT`]( https://conda.io/docs/building/environment-vars.html#environment-variables-set-during-the-build-process ). Though that includes the `.`. Correct @jschueller?